src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -2012,7 +2059,7 @@ public bool MapCurrentSrcToCompileTimeSrc(string currentSrc, out string compiler
                             continue;   // match didn't end at a directory separator, not actually a match
                         }
                         compilerSrc = Path.Combine(e.CompileTimePath, file);    // map to the compiled location
-                        if (compilerSrc.IndexOf('\\') < 0)
+                        if (compilerSrc.IndexOf('\\') > 0)
```

Comment:
@paulmaybee I think this was a logic error, but I wanted to check to see if there was a case I wasn't considering that this was trying to catch.